SP_aircraft.py

```diff
@@ -345,33 +345,33 @@ def run_optimal_737(objective, fixedBPR, pRatOpt = False):
     
     substitutions = get737_optimal_subs()
 
-    if Nmission > 1:
+    if Nmission == 2:
         substitutions.update({
-            'R_{req}': [3000.*units('nmi'),2000.*units('nmi')],
-            'n_{pass}': [180., 180.],
+            'R_{req}': [3000.*units('nmi'),3500.*units('nmi')], #,2500.*units('nmi')
+            'n_{pass}': [180., 160.], #, 140.
```

Comment:
@1ozturkbe for the 737 lets keep these subs what we have in the paper

Reply:
Sure.